cudaaligner/include/claragenomics/cudaaligner/alignment.hpp

```diff
@@ -24,11 +24,16 @@ namespace cudaaligner
 /// \addtogroup cudaaligner
 /// \{
 
-/// \typedef FormattedAlignment
-/// Holds formatted strings representing an alignment.
-/// FormattedAlignment.first = formatted string for query
-/// FormattedAlignment.second = formatted string for target
-typedef std::pair<std::string, std::string> FormattedAlignment;
+/// FormattedAlignment -Holds formatted strings representing an alignment.
+typedef struct FormattedAlignment
+{
+    /// \brief FormattedAlignment.first = formatted string for query
+    std::string first;
+    /// \brief FormattedAlignment.pairing = formatted pairing string
+    std::string pairing;
+    /// \brief FormattedAlignment.first = formatted string for target
+    std::string second;
+} FormattedAlignment;
 
```

Comment:
Since we use full struct now, I would suggest to add a
`std::ostream& operator << (std::ostream& os, FormattedAlignment const& formattedalignment)` for output.

There is one problem with output to the terminal: the output is garbled when the width is longer than the terminal width.
I would suggest to add a `linebreak_after` member to FormattedAlignment with default `0`, i.e. no line breaks, and use that in the `<<` operator.

Opinions? @ohadmo @tijyojwad

Reply:
I agree